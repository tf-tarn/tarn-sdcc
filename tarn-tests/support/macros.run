./sim/sim -q -h -s 100000 $program_file > $output
