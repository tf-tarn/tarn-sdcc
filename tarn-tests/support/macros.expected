OUTPUT=30
OUTPUT=31
OUTPUT=32
OUTPUT=33
OUTPUT=30
OUTPUT=31
OUTPUT=40
OUTPUT=31
OUTPUT=88
OUTPUT=99
OUTPUT=41
OUTPUT=30
OUTPUT=88
OUTPUT=99
OUTPUT=40
OUTPUT=31
OUTPUT=41
OUTPUT=30
OUTPUT=40
OUTPUT=31
OUTPUT=84
OUTPUT=95
OUTPUT=41
OUTPUT=30
OUTPUT=84
OUTPUT=95
OUTPUT=40
OUTPUT=31
OUTPUT=83
OUTPUT=94
OUTPUT=41
OUTPUT=30
OUTPUT=83
OUTPUT=94
OUTPUT=10
OUTPUT=04
OUTPUT=21
OUTPUT=01
OUTPUT=50
OUTPUT=40
OUTPUT=51
OUTPUT=2f
OUTPUT=40
OUTPUT=31
OUTPUT=41
OUTPUT=30
OUTPUT=00
OUTPUT=00
OUTPUT=00
OUTPUT=00
OUTPUT=00
OUTPUT=00
OUTPUT=20
OUTPUT=22
OUTPUT=21
OUTPUT=03
OUTPUT=10
OUTPUT=00
OUTPUT=00
OUTPUT=fd
OUTPUT=ff
OUTPUT=03
